axelrod/fingerprint.py

```diff
@@ -327,14 +327,14 @@ def fingerprint(
         self.data = generate_data(self.interactions, self.points, edges)
         return self.data
 
-    def plot(self, col_map: str = 'seismic', interpolation: str = 'none',
+    def plot(self, cmap: str = 'seismic', interpolation: str = 'none',
              title: str = None, colorbar: bool = True,
              labels: bool = True) -> plt.Figure:
         """Plot the results of the spatial tournament.
 
         Parameters
         ----------
-        col_map : str, optional
+        cmap : str, optional
```

Comment:
Based on the fact that `mpl.imshow` uses `cmap` right (https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.imshow.html)?

I'm happy with this (should have gone with this from the start really), just checking.